saritasa_invocations/django.py

```diff
@@ -207,3 +212,97 @@ def set_default_site(context: invoke.Context) -> None:
             "set_default_site --name localhost:8000 --domain localhost:8000"
         ),
     )
+
+
+@invoke.task
+def load_db_dump(context: invoke.Context, file: str = "") -> None:
+    """Reset db and load db dump."""
+    django.resetdb(context, apply_migrations=False)
```

Comment:
This call (`django.resetdb`) looks confusing because `resetdb` is defined in this file.
Checkout imports in this file - it contains `from . import django`